examples/run_field_dep_justMw.py

```diff
@@ -1,151 +1,191 @@
-''' Field Sweep at constant power
+""" Field Sweep at constant power
 =================================
 Here we will perform a series of echoes at a range of designated field values. This is normally run at a power of 3-4 W. To run this experiment, please open Xepr on the EPR computer, connect to spectrometer, enable XEPR_API. Then, in a separate terminal, run the program XEPR_API_server.py, wait for it to tell you 'I am listening' - then, you should be able to run this program in sync with the power_control_server.
 To run this in sync with the power_control_server, open a separate terminal on the NMR computer and move into git/inst_notebooks/Instruments and run winpty power_control_server(). This will print out "I am listening" when it is ready to go. You can then proceed to run this script to collect your field sweep data
-'''
-
+"""
 from pylab import *
 from pyspecdata import *
 import os
-import sys
 import SpinCore_pp
-from SpinCore_pp/ppg import run_spin_echo
+from SpinCore_pp.ppg import run_spin_echo
 from datetime import datetime
 import numpy as np
-from Instruments import Bridge12,prologix_connection,gigatronics
-from serial import Serial
+from Instruments import power_control, Bridge12, prologix_connection, gigatronics
 from Instruments.XEPR_eth import xepr
 import h5py
+
 fl = figlist_var()
 mw_freqs = []
-#{{{these parameters change for each sample
-field_axis = r_[3422:3426:.5]
-print("Here is my field axis:",field_axis)
-output_name = 'TEMPOL_heat_exch_289uM_field_dep'
-node_name = 'field_sweep_1'
-adcOffset = 28
-gamma_eff = (14.549013/3424.42)
-nScans = 1
-nEchoes = 1
+field_axis = r_[3512:3518:2.0]
```

Comment:
should we generate this using ppt and the nu_ESR and gamma_eff?:
`left = (config_dict['ppt']*config_dict['nu_ESR']*(config_dict['gamma_eff']^-1)) - 3`
`right = (config_dict['ppt']*config_dict['nu_ESR']*(config_dict['gamma_eff']^-1)) + 3`
`field_axis = r_[left: right: 1.0]`